T/tree_sitter_highlight/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "tree_sitter_highlight"
+version = v"0.20.6"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/tree-sitter/tree-sitter/",
+              "ccd6bf554d922596ce905730d98a77af368bba5c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+ls
+cd $WORKSPACE/srcdir/
+
+# Use cargo-rustc to build tree-sitter-highlight as a dynamic library here vvvvvv
```

Comment:
`vvvvvv`?

Reply:
This was to point to cdylib :arrow_double_down: